### DISCUSSION TITLE
Like theme, Is there a way to change font style?

### DISCUSSION THREAD

**Discussion by sairaamv** [h2oai/wave#1010] [Q&A] [answered]
I could see the theme in `ui/src/theme.ts` and could add my custom theme format. Thanks to that
Is there a way to edit fontstyle?

Comments/thoughts appreciated
TIA

**Comment by mturoci** [comment, marked as answer]
Hi @sairaamv.

I am not sure I fully understand what you mean. `ui/src/theme.ts` cannot be edited directly via python. The only way to toggle theme currently is via `meta_card`.

Font is specified in https://github.com/h2oai/wave/blob/813d334f85ae821042a368e2d3f30744d8b5f534/ui/src/index.scss#L18. Although we don't support custom fonts natively, it should be very easy using newly added api for external stylesheets.

See:
https://github.com/h2oai/wave/blob/master/py/examples/meta_inline_stylesheet.py
https://github.com/h2oai/wave/blob/master/py/examples/meta_stylesheet.py

**Reply by sairaamv** [reply]
Thats awesome, thanks
